openstf/model/regressors/linear.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from openstf.model.metamodels.missing_values_handler import MissingValueHandler
+import numpy as np
+from sklearn.linear_model import LinearRegression
+
+from openstf.model.regressors.regressor import OpenstfRegressor
+
+
+class _LinearRegressor(LinearRegression):
+    """Wrapper around sklearn linearRegression,
+    that provides features importances as absolute values of the coefficients."""
+
+    def __init__(self):
+        super().__init__()
+
+    def fit(self, X, y):
+        super().fit(X, y)
+        self.feature_importances_ = np.abs(self.coef_)
+
+
+class LinearRegressor(MissingValueHandler):
+    def __init__(self, missing_values=np.nan, imputation_strategy=None, fill_value=0):
+        super().__init__(
+            _LinearRegressor(),
+            missing_values=missing_values,
+            imputation_strategy=imputation_strategy,
+            fill_value=fill_value,
+        )
+
+
+class LinearOpenstfRegressor(LinearRegressor, OpenstfRegressor):
```

Comment:
It took me a while to understand all the inheritance that is going on in this file. I think for now it is fine but can you think of a way simplify this in the future?

Reply:
LinearRegressor is the sklearn-like class (wrapped in the metamodel).
LinearOpenstfRegressor inherits from this sklearn-like class and implements the OpenstRegressor interface.
The weird class _LinearRegression has been removed because feature_importance calculation has been moved in LinearOpenstfRegressor. 
It seems to me that now the inheritance is quite similar to lgbm or xgb models.

Reply:
Seems much more clear, thank you for adding these clarifications!